### PR TITLE
[DEV-2784], [DEV-2752], [DEV-2889], [DEV-2866], [NO-TASK], bundle optimizations

### PR DESCRIPTION
**Pull request by lapatric**
- [DEV-2784] add payment routes
- [NO-TASK] Lazy load all screens
- [NO-TASK] Added analyzer
- [NO-TASK] Switched to JPGs
- [DEV-2752] integrate info banner
- [DEV-2889] remove copy symbol from beneficiary name
- [DEV-2866] fix title text overflow
- [NO-TASK] send close message on card TX

[DEV-2784]: https://dfxswiss.atlassian.net/browse/DEV-2784?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DEV-2752]: https://dfxswiss.atlassian.net/browse/DEV-2752?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DEV-2889]: https://dfxswiss.atlassian.net/browse/DEV-2889?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DEV-2866]: https://dfxswiss.atlassian.net/browse/DEV-2866?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/screens/payment-routes.screen.tsx
Comment (davidleomay) [inline src/screens/payment-routes.screen.tsx:1]: Add a loading spinner

Comment (davidleomay) [reply src/screens/payment-routes.screen.tsx:1]: Order routes by blockchain?

File: src/screens/payment-routes.screen.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+              {routes.sell.map<JSX.Element>((route) => (
+                <StyledDataTableExpandableRow
+                  label={`${route.currency.name} / ${route.iban}`}
+                  expansionItems={[
+                    { label: translate('screens/payment', 'Currency'), text: route.currency.name },
+                    { label: translate('screens/payment', 'IBAN'), text: route.iban },
+                    {
+                      label: translate('screens/payment', 'Deposit address'),
+                      text: route.deposit.address,
+                    },
```
Comment (davidleomay) [inline src/screens/payment-routes.screen.tsx:79]: Add deposit blockchains here as well

File: src/screens/payment-routes.screen.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+              {routes.swap.map<JSX.Element>((route) => (
+                <StyledDataTableExpandableRow
+                  label={route.asset.name}
+                  expansionItems={[
+                    { label: translate('screens/payment', 'Asset'), text: route.asset.name },
+                    { label: translate('screens/home', 'Blockchain'), text: route.asset.blockchain },
+                    {
+                      label: translate('screens/payment', 'Deposit address'),
+                      text: route.deposit.address,
+                    },
```
Comment (davidleomay) [inline src/screens/payment-routes.screen.tsx:108]: Add deposit blockchains here as well

File: src/screens/payment-routes.screen.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+                </StyledDataTableExpandableRow>
+              ))}
+            </StyledDataTable>
+          )}
+          {routes?.swap.length && (
+            <StyledDataTable label={translate('screens/payment', 'Swap')}>
+              {routes.swap.map<JSX.Element>((route) => (
+                <StyledDataTableExpandableRow
+                  label={route.asset.name}
+                  expansionItems={[
```
Comment (davidleomay) [inline src/screens/payment-routes.screen.tsx:102]:
```ts
label={`${route.asset.blockchain} / ${route.asset.name}`}
```

File: src/screens/payment-routes.screen.tsx (hunk 1 above)
Comment (lapatric) [reply src/screens/payment-routes.screen.tsx:79]: Is it not already there on l. 90?

```ts
<p className="text-right overflow-ellipsis">
  {route.deposit.blockchains.map((blockchain: Blockchain) => toString(blockchain)).join(', ')}
</p>
```

Comment (lapatric) [reply src/screens/payment-routes.screen.tsx:79]: I added it inside the dropdown too, makes sense to have it labeled xD 👍

**Review by davidleomay — APPROVED** [review]

**Pull request merged**